### DISCUSSION TITLE
Setup Application with two services that share the same configmap

### DISCUSSION THREAD

**Discussion by alexBonadei** [mia-platform/community#443] [Q&A] [answered]
### What product are you having troubles with?

Marketplace

### What Console version are you using?

1.98.6

### Description

Hi, I'm setting up a new application that is composed of two services.

I want to setup a default configmap that is used by the two services.
I tried like this:
```json
{
  ...,
  "resources": {
    "services": {
      "firstService":{
        ...,
        "defaultConfigMaps": [
          {
              "name": "my-configmap",
              "mountPath": "/config",
              "files": [
                  {
                      "content": "...content of the configmap...",
                      "name": "config.json"
                  }
              ],
              "viewAsReadOnly": false
          }
        ]
      },
      "secondService":{
        ...,
        "defaultConfigMaps": [
          {
              "name": "my-configmap",
              "mountPath": "/config",
              "files": [
                  {
                      "name": "config.json"
                  }
              ],
              "viewAsReadOnly": false
          }
        ]
      }
    }
  }
}
```

This doesn't work because (during the application setup) during the setup of the second service there will be an error because the name of the configmap was already in use.

What is the proper way to setup only one configmap and mount it on both services? If it's possible.

Thanks in advance

**Comment by fredmaggiowski** [comment, marked as answer]
Hi

As of now I think there is no way since, as you noticed, the frontend returns an error upon creation. I'd open a feature request in the internal board :)

**Reply by alexBonadei** [reply]
Yes, I will! Thanks!
